pydatastructs/graphs/tests/test_adjacency_matrix.py

```diff
@@ -26,6 +26,5 @@ def test_AdjacencyMatrix():
     assert neighbors == [v_1]
     g.remove_edge(0, 1)
     assert g.is_adjacent(0, 1) is False
-
-    assert raises(ValueError, lambda: g.add_edge('u', 'v'))
-    assert raises(ValueError, lambda: g.add_edge('v', 'x'))
```

Comment:
Why did you remove these tests?

Reply:
I thought that these tests were in a form that had to be edited (in accordance with the other tests I saw in the tests.py file), which I tried to accomplish by my tests.